.github/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 <h3>New features since last release</h3>
 
+* Add validation for noise channel parameters, now it raises an `ValueError` when
+  noise parameters are invalid.
```

Comment:
Don't forget to add a link back to your PR here!
```suggestion
* Added validation for noise channel parameters. Invalid noise parameters now
   raise a `ValueError`.
  noise parameters are invalid.
```

Reply:
Yep, forgot about that 😅